docs/pages/submit/introduction.mdx

```diff
@@ -15,6 +14,25 @@ import { BoxLink } from '~/ui/components/BoxLink';
 
 It makes submitting your apps simple and easy by creating and handling submission credentials and running the submission process. You can trigger a submission from a CLI command, after a build is finished, or from a CI/CD service. It's the fastest way to take Android and iOS builds and submit them to the app stores.
 
+### What does "Submit" mean in EAS?
+
+**Submit** is not the same as releasing your app directly to production. Instead, it delivers your app to the app stores' distribution pipelines (a chosen track on Google Play or TestFlight for iOS), following the [default submission behavior for app stores](/build/automate-submissions/#default-submission-behavior-for-app-stores).
```

Comment:
```suggestion
**EAS Submit** delivers your app to the app stores' distribution pipelines (a chosen track on Google Play or TestFlight for iOS), following the [default submission behavior for app stores](/build/automate-submissions/#default-submission-behavior-for-app-stores). It queues up your app for distribution on the Play Store Console and App Store Connect, and then you can log into those sites to send your apps off to review, so then they can be distributed to your users.
```

Reply:
@amandeepmittal I was thinking of how to phrase / frame this in a sense of "what does EAS Submit do" vs. "what does "Submit" mean" ... what do you think?